app/db/models/law_policy/geography.py

```diff
@@ -11,7 +11,7 @@ class Geography(Base):  # noqa: D101
     id = sa.Column(sa.Integer, primary_key=True)
     # to display to end-users
     display_value = sa.Column(sa.Text, unique=True, nullable=False)
-    slug = sa.Column(sa.Text, unique=True, nullable=False, index=True)
+    slug = sa.Column(sa.Text, nullable=False, unique=True, index=True)
```

Comment:
Just looks like a pre-commit refactor?

Reply:
yeah, I thought I'd undone a change - the alembic generated migration was wrong & I was trying to get it to generate something valid. In the end I just had to re-order some of the migration lines manually :(